kafka-connect-neo4j/docker/contrib.sink.string-json.neo4j.json

```diff
@@ -12,6 +12,6 @@
     "neo4j.authentication.basic.username": "neo4j",
     "neo4j.authentication.basic.password": "connect",
     "neo4j.encryption.enabled": false,
-    "neo4j.topic.cypher.my-topic": "MERGE (p:Person{name: event.name, surname: event.surname}) MERGE (f:Family{name: event.surname}) MERGE (p)-[:BELONGS_TO]->(f)"
+    "neo4j.topic.cypher.my-topic": "ME RGE (p:Person{name: event.name, surname: event.surname}) MERGE (f:Family{name: event.surname}) MERGE (p)-[:BELONGS_TO]->(f)"
```

Comment:
beautiful.  I love this because this is a great example of what a user would actually screw up.  I did it myself a bunch of times.   Simple cypher errors in your config is (my most hated) error with this plugin.

Reply:
ups this was just for testing :) shouldn't have made it in :)